backend/src/api/users.ts

```diff
@@ -83,17 +83,6 @@ export const update = wrapHandler(async (event) => {
     user.lastName = body.lastName ?? user.lastName;
     user.fullName = user.firstName + ' ' + user.lastName;
     await User.save(user);
-
-    if (body.organization) {
```

Comment:
Why is this no longer needed?

Reply:
I removed the ability for users to join organizations themselves (public organizations) since I don't think we would ever use this. Instead, organizations may invite users to join their organization.